update LTS (#184)
AzureAD/microsoft-identity-abstractions-for-dotnet: README.md

Microsoft.Identity.Abstractions contain interfaces and POCO classes used in the Microsoft .NET authentication
libraries (Microsoft.IdentityModel, MSAL.NET and Microsoft.Identity.Web). It exposes concepts in three domains:

> ## Nuget package

Reviewer: ```suggestion
## NuGet Package
```

[![NuGet](https://img.shields.io/nuget/v/Microsoft.Identity.Abstractions.svg?style=flat-square&label=nuget&colorB=00b200)](https://www.nuget.org/packages/Microsoft.Identity.Abstractions/)

## Version Lifecycle and Support Matrix

See [Long Term Support policy](./supportPolicy.md) for details.

> The following table lists IdentityWeb versions currently supported and receiving security fixes.

Reviewer: ```suggestion
The following table lists Microsoft.Identity.Abstractions versions currently supported and receiving security fixes.
```

> | Major Version | Last Release | Patch release date  | Support phase|End of support |

Reviewer: ```suggestion
| Major Version | Last Release | Patch Release Date  | Support Phase | End of Support |
```
